gmmp/management/commands/bulk_password_reset_emails.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.core.mail import send_mass_mail
+from django.template.loader import render_to_string
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        users = User.objects.all()
+        datatuple = []
+        password_reset_form = render_to_string(
+            'emails/welcome_password_reset.html', )
+        for user in users:
+            datatuple.append(
+                (
+                    'Reset your password on GMMP',
+                    password_reset_form,
+                    'from@gmmp.com',
```

Comment:
not sure if this should be `from@gmmp.com`

Reply:
I know! Still working on rendering the email correctly first as that takes more priority...